### DISCUSSION TITLE
integração com meu aplicativo

### DISCUSSION THREAD

**Discussion by brunowilliang** [enricodias/smsdev-php#18] [General]
Oie, tudo bem? existe alguma forma eu integrar na área do cliente pra ele comprar os creditos e utilizar?

**Comment by enricodias** [comment]
Não, isso não está na API da smsdev.

**Reply by brunowilliang** [reply]
Certo, @enricodias  te mandei um email no [suporte@smsdev.com.br](mailto:suporte@smsdev.com.br) com um problema, acabei de comprar algumas msgs mas não está enviando do jeito que eu preciso, alguem de vocês consegue entrar em contato comigo?

**Reply by enricodias** [reply]
Eu não trabalho na smsdev, eu apenas desenvolvi essa lib para consumir a API deles.
